status.go

```diff
@@ -167,9 +181,10 @@ type statusDeserializationFunc func(map[string]interface{}) (StatusPoint_, error
 
 var statusDeserializationFuncs = map[int]statusDeserializationFunc{
 	1: importStatusV1,
+	2: importStatusV2,
```

Comment:
Sorry, I should have pointed you at this yesterday, but there's a cleaner way of organising the multi-version code with less boilerplate - it means you just have a map of the the fields functions and no repetitive importStatusVx funcs. You can see an example in subnet.go. Would you mind making this work like that, if it's not too painful?